tests/test_shell.py

```diff
@@ -199,14 +323,233 @@ def test_remove(self):
         assert job_to_remove in project
         assert job_to_remove.doc.a == 0
         assert len(job_to_remove.doc) == 1
-        with pytest.deprecated_call():
```

Comment:
Are those calls not considered deprecated anymore?

Reply:
The deprecated call was because of `get_id` function. I removed it so that it's not needed anymore.